buildroot-external/package/pi-bluetooth/hciuart.service

```
@@ -5,6 +5,7 @@ After=dev-serial1.device
 
 [Service]
 Type=forking
+ExecCondition=/bin/sh -c "if [ "$(cat /proc/device-tree/$(cat /proc/device-tree/aliases/bluetooth)/bluetooth/status)" == "okay" ]; then exit 1; fi"
```

Wouldn't there be a problem with escaping? Also could be simplified:
```suggestion
ExecCondition=/bin/sh -c '[ "$(cat /proc/device-tree/$(cat /proc/device-tree/aliases/bluetooth)/bluetooth/status)" != "okay" ]'
```

---

It did work on the shell but I wanted to double check. I considered single quote, but somehow I thought this would prevent `$()` from working. But that is not the case. So I prefer your solution. Thanks!